wave_optimization/CMakeLists.txt

```diff
@@ -5,38 +5,50 @@ IF (CMAKE_COMPILER_IS_GNUCXX)
   ADD_DEFINITIONS(-std=gnu++0x)
 ENDIF()
 
+# PACKAGES
+FIND_PACKAGE(Ceres REQUIRED)
+
 # INCLUDES
```

Comment:
Is there a way to not depend on Ceres unless you select it to be used? We don't want 100 dependencies to install, in general.  I noticed this with MCPTAM too, still depends on g2o even though Ceres was being used (or vice versa).